### PR TITLE
Use print() function in both Python 2 and Python 3

### PR DESCRIPTION
**Pull request by cclauss**
Legacy __print__ statements are syntax errors in Python 3 but __print()__ function works as expected in both Python 2 and Python 3.

A _focused subset_ of #6983 which currently has lots of conflicts.  This PR contains only one kind of change to increase the likelihood of rapid review.

Thank you for submitting a pull request!

* [x] If this is your first PR, make sure to add yourself to AUTHORS.

### CODE REVIEW

File: tests/bullet/Demos/HelloWorld/parse.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```
Comment (sbc100) [inline tests/bullet/Demos/HelloWorld/parse.py]: can you put this below the copyright with the other import?

File: tests/fuzz/creduce_tester.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 """Usage: creduce ./creduce_tester.py newfail1.c
 """
+from __future__ import print_function
```
Comment (sbc100) [inline tests/fuzz/creduce_tester.py]: Move this down to be with the other imports?

File: tools/update_js.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 '''
 Performs a search-replace in all of js/
 '''
+from __future__ import print_function
```
Comment (sbc100) [inline tools/update_js.py]: Move down one line.

**Comment by cclauss** [comment]
Should I be concerned about these test failures??

**Comment by sbc100** [comment]
No, there is some flakeyness with the browser tests.  I'll re-run time for you

File: tests/sockets/socket_relay.py [hunk 4]
```diff
@@ -13,6 +13,7 @@
 and then make a connection to another port, and do bidirectional
 communication. We need to actually listen on both ports.
 """
+from __future__ import print_function
```
Comment (sbc100) [inline tests/sockets/socket_relay.py]: This one still needs moving.

**Comment by kripken** [comment]
The other failures look like old temporary breakage we had on master. Given the contents here I think this is safe to land. Thanks again @cclauss!

**Pull request merged**